lib/vending_server.ex

```diff
@@ -1,55 +1,55 @@
-defmodule VendingMachine.VendingServer do
-  use GenServer
-  alias VendingMachine.{VendingServer, ItemController, PaymentController, Stash}
+# defmodule VendingMachine.VendingServer do
+#   use GenServer
```

Comment:
Why this commented file is still there?

Reply:
Planning to use this server again after adding the database.